assets/queries/openAPI/schema_enum_invalid/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	[path, value] := walk(doc)
+
+	openapi_lib.invalid_field(value.enum[x], value.type)
```

Comment:
I think we should add an check if `type` is defined

Reply:
It is not necessary. The '`value.type`' already check